nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/core/ElasticSearchAdminImpl.java

```diff
@@ -93,12 +99,27 @@
 
     /**
      * Init the admin service, remote configuration if not null will take precedence over local embedded configuration.
+     * @deprecated since 9.1
```

Comment:
Explain what to use instead: `@deprecated since 9.1, use XXX instead`.

Also add a `@Deprecated` annotation on the constructor.